_data/simple-icons.json

```
@@ -13220,6 +13220,11 @@
             "hex": "026938",
             "source": "https://www.saudiatechnic.com/media-center/press-kit"
         },
+        {
+            "title": "Saxo Bank",
+            "hex": "111827",
+            "source": "https://www.home.saxo/-/media/global/logos/saxo-2022/saxo-beinvested-logo-white.svg"
```

We should use the following links:
```suggestion
            "source": "https://www.home.saxo/about-us/media-center",
            "guidelines": "https://www.home.saxo/about-us/media-center"
```